script/DeployBase.s.sol

```diff
@@ -124,6 +124,7 @@ struct DeployParams {
     uint256 attestationsWeight;
     uint256 blocksWeight;
     uint256 syncWeight;
+    uint256 defaultAllowedExitDelay;
```

Comment:
If you want to have default word here, also add it to the other params

Reply:
not really but it's a good notice